### PR TITLE
Filter Style

### PR DESCRIPTION
**Pull request by MatsJoh50**
Added styln and reactive interferance with media querys for the filter part.

upcomming changes:
continue with the close "cross"
add header+footer in htm

### CODE REVIEW

**Comment by EmilioSanchezLernia** [comment]
This looks really good so far! Keep up the great work 💯

File: modules.js
Comment (EmilioSanchezLernia) [inline modules.js:1]: It might be wiser to remove this part of the pull request for now and make another pull request with just the modules to make it a little cleaner

File: src/filter.scss [hunk 1]
```diff
@@ -1,59 +1,75 @@
...
+    margin: 10px auto;
     border: 1px solid gray;
-    width: 80%;
-    max-width: 1000px;
+    width: 91vw;
+    // max-width: 1000px;
     max-height: fit-content;
     display: flex;
     flex-direction: column;
+    animation: openUp 0.2s;
```
Comment (AnnaLenaO) [inline src/filter.scss:15]: Nicely done! Good initiative that we have the same type of animation throughout the webbsite.

**Review by AnnaLenaO — COMMENTED** [review]
Nicely done! Good initiative that we have the same type of animation throughout the webbsite.

**Review by EmilioSanchezLernia — COMMENTED** [review]
Looking at the rest of the code it looks good 👍

**Review by EmilioSanchezLernia — APPROVED** [review]
Everything looks good, I can't see any errors. Great job man!
